docs/xocto/development.md

```diff
@@ -46,6 +46,15 @@ of this workflow:
 This is why typically in shared projects version releases are separated into
 their own pull requests.
 
+## Altering dependencies
+
+* Edit `pyproject.toml`.
+* `make sync_requirements` to refresh `requirements.txt`.
+* `make install test format_check lint_check mypy`.
```

Comment:
It's probably worth copying the makefile targets from the CookieCutterPackage repo:
https://github.com/octoenergy/cookiecutter-kraken-package/blob/main/%7B%7B%20cookiecutter.repo_name%20%7D%7D/makefile